spec/adapters/postgres_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'spec_helper'
+
+describe Polo::Adapters::Postgres do
+  include Polo::Adapters::Postgres
+
+  let(:netto) do
+    AR::Chef.where(name: 'Netto').first
+  end
+
+  before(:all) do
+    TestData.create_netto
+  end
+
+  let(:translator) { Polo::SqlTranslator.new(netto, Polo::Configuration.new(adapter: :postgres)) }
+
+  describe '#ignore_transform' do
+    it do
```

Comment:
I don't really get what's going on here 😅
So I think a good title for this test would go a long way.